pkg/cloud/endpoints/endpoints.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2020 The Kubernetes Authors.
```

I know it's my copy-pasta, soz.